shared/googleCloudUtils.ts

```diff
@@ -127,45 +154,59 @@ class Cloud {
       };
     }
   }
+  static async downloadUnSyncedBackupFiles(): Promise<Boolean> {
```

Comment:
Do we have to also sync the injibackup ZIP file across iOS devices? As per the official docs of [downloadFile](https://react-native-cloud-storage.vercel.app/docs/api/CloudStorage#downloadfilepath-scope) the downloadFile function downloads the file in full even if the file has been synced as has the suffix `.icloud`, in this case should we turn off the sync in the backup directory and only let the state machines manage the state of the ZIPs & .injibackup files ?


**Tl; dr** : Should we turn off the iCloud sync for the `inji/` & `mmkv` directory across iOS devices, will they ever conflict with the state of each other devices? Two-way download/syncs can cause issues in the longer run.

**Background** : I'm implementing a feature which looks for `.injibackup` files and then subsequently restores the files on app launch. This might cause some issues if a person uses the wallet from multiple devices and backs up in one and restores in another.

Reply:
This call is just for the sync only. This method make sures that the .icloud file has been synced properly and on next call to download the .zip file will exist. 

Also yes this method can be used to sync files across the devices with same apple ID logged in. 

Inorder to implement the first launch we still need a sync. Thats why this method was introduced.